pandas_datareader/google/daily.py

```diff
@@ -29,6 +29,13 @@ class GoogleDailyReader(_DailyBaseReader):
         requests.sessions.Session instance to be used
     """
 
+    def __init__(self, symbols=None, start=None, end=None, retry_count=3,
```

Comment:
This isn't necessary.  But if it were, the arguments should have the same order.